src/box/box.scss

```diff
@@ -144,7 +144,7 @@
 
     // Row dragging styles
     &.sortable-chosen {
-      background-color: var(--color-bg-primary-light);
+      background-color: var(--color-bg-primary);
     }
```

Comment:
`--color-bg-primary-light` doesn't exist so I replaced it with `--color-bg-primary` as a guess 🤷‍♂️ 

It's hard to confirm if this is the right replacement because I couldn't find any documentation on the `.sortable-chosen` class.

Reply:
/cc @auareyou and @simurai